### PR TITLE
Windows Can Now Official Swap With Each Other

### PR DESCRIPTION
**Pull request by rhorton5**
# Program.cs
Changed how the process swapping takes place.  It still will close a process and open a new one, but it uses `CurrentForm` as a parameter.  If `CurrentForm` is returned as `null`, then the program will close itself.  For swapping Windows on another Form, you'll need to specify which one is being opened.  (i.e. `Program.CurrentForm = new SpecimenForm();` to get SpecimenForm to run.

# SpecimentForm.cs, TemplateForm.cs, SpecimenSummaryForm.cs
All of these windows can now swap between different windows using a green arrow symbol towards the right side of the navigation strip (or the menu strip for SpecimenSummaryForm).

### CODE REVIEW

**Comment by rhorton5** [comment]
Sorry, I forgot I had this in the sprints.  There is now a save confirmation for both SpecimenForm and TemplateForm.  I did notice it will make this MessageBox appear when you've just entered the templates or specimen forms and then exited, but that might just be due to the `isUpdated()` function.

**Comment by NicholasJohnsonDev** [comment]
Staring the review

File: Team 6 Senior Project/SpecimensForm.cs [hunk 1]
```diff
@@ -596,5 +609,33 @@ private void specimensDataGridView_DataError(object sender, DataGridViewDataErro
...
+            this.Close();
+        }
+
+        private void summaryToolStripMenuItem_Click(object sender, EventArgs e)
+        {
+            Program.CurrentForm = new SpecimensSummaryForm();
+            this.Close();
+        }
+
+        private void toolStripDropDownButton1_Click(object sender, EventArgs e)
```
Comment (NicholasJohnsonDev) [inline Team 6 Senior Project/SpecimensForm.cs:625]: 625-634 don't do anything and can be deleted. Be sure to remove the matching references in the code behind.

**Comment by NicholasJohnsonDev** [comment]
Bug with isUpdated() in each of the [forms].cs but existed before this branch (I definitely made this bug), just wanted to note it somewhere.

Left some empty event functions, but doesn't cause any issues so we can get rid of them later.

The form swapping feature works great! Approved.

**Pull request merged**